### DISCUSSION TITLE
This licence is tied to another Craft install – on local/staging envs

### DISCUSSION THREAD

**Discussion by RyanRoberts** [craftcms/cms#11464] [Q&A] [answered]
I've just launched a site which is now live and has fully paid up Craft and plugin licenses. However, when I login to Craft locally or on a whitelisted staging domain I get a red banner message and the following warnings in the plugins list.

https://media.yep.digital/KZEPsX

How can I resolve this warning? It's only plugins, Craft is ok.

**Comment by brandonkelly** [comment, marked as answer]
That happens if your plugin licenses (stored in the project config) are associated with a different Craft license key (stored at `config/license.key`).

Most likely, your `config/license.key` files are not consistent across all of your environments. Figure out which license key the plugins were actually purchased with, and copy its license file over to your other environments (or just start tracking it in Git).

**Reply by RyanRoberts** [reply]
Ah I see, so the plugins are looking for the main license key. I'll make sure that's synced correctly and check again 👍
